<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccessPage</class>
 <widget class="QWidget" name="AccessPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>691</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,5,5">
   <item>
    <layout class="QHBoxLayout" name="queryLayout" stretch="3,1">
     <item>
      <widget class="QComboBox" name="combo_area">
       <item>
        <property name="text">
         <string>TWY A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TWY B</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_query">
       <property name="text">
        <string>설정 상태 조회</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>현재 설정 상태</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_status">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row/>
     <column>
      <property name="text">
       <string>구역</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>인원 제한 등급</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>차량 제한 등급</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>TWY XX</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>3</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_limit_setting">
     <property name="title">
      <string>제한 설정</string>
     </property>
     <layout class="QVBoxLayout" name="groupLayout" stretch="1,1,1,1">
      <item>
       <layout class="QHBoxLayout" name="targetLayout" stretch="1,1,1,3">
        <item>
         <widget class="QLabel" name="label_target">
          <property name="text">
           <string>제한 대상</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chk_person">
          <property name="text">
           <string>인원</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chk_vehicle">
          <property name="text">
           <string>차량</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="gradeLayout" stretch="1,1,1,1,2">
        <item>
         <widget class="QLabel" name="label_grade">
          <property name="text">
           <string>제한 등급</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_grade1">
          <property name="text">
           <string>1 등급</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_grade2">
          <property name="text">
           <string>2 등급</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_grade3">
          <property name="text">
           <string>3 등급</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="reasonLayout" stretch="1,6">
        <item>
         <widget class="QLabel" name="label_reason">
          <property name="text">
           <string>사유</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_reason">
          <item>
           <property name="text">
            <string>활주로 보수 공사</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>기타</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="registerLayout" stretch="4,1">
        <item>
         <spacer name="spacer1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_register">
          <property name="text">
           <string>조건 등록</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
